src/Sentry/ISentryClient.cs

```diff
@@ -22,6 +22,12 @@ public interface ISentryClient
         /// <returns>The Id of the event.</returns>
         SentryId CaptureEvent(SentryEvent evt, Scope? scope = null);
 
+        /// <summary>
+        /// Captures a manually created user feedback and sends it to Sentry.
+        /// </summary>
+        /// <param name="userFeedback">The user feedback to send to Sentry.</param>
+        void CaptureUserFeedback(SentryUserFeedback userFeedback);
```

Comment:
Can we add some extension methods to `SentryClientExtensions`?

Calling `CaptureBla(new Bla(ble, ble ble)` is really not a nice API.

Proposed overload:
`public static void CaptureUserFeedback(SentryId eventId, string email, string comments, string name = null);`

Assuming only eventid, email and comments are required by the server.

Reply:
Extension CaptureUserFeedback(this ISentryClient client, UserFeedback userFeedback)
Calls 
CaptureUserFeedback(SentryId sentryId, string email, string comments, string? name = null)

with that, I'll change the Envelope.FromUserFeedback to not receive the UserFeedback but email, name ,... Otherwise, I will have to create another UserFeedback object

Reply:
Unless, you propose to edit SentrySdk.cs so devs would call CaptureUserFeedback(id, email,..) instead of CaptureUserFeedback(new ,,,) and internally the SDK is going to create the UserFeedback.

Reply:
Keep the `UserFeedback` as the parameter. Make the extension be the overload with all those arguments